src/app/screens/swap/useSwap.test.ts

```diff
@@ -0,0 +1,129 @@
+import { Currency } from 'alex-sdk';
+
+// TODO tim: fix jest transpiler to be able to import xverse-core
```

Comment:
for the initial release , do we want to go with the copied function or is this in progress?

Reply:
let me make a follow up task to fix this test file, but essentially, I was testing a duplicated function to help with TDD.

Reply:
https://linear.app/xverseapp/issue/ENG-2474/fix-useswap-test-file